.gitignore

```diff
@@ -34,3 +34,6 @@ builds/
 
 .idea/*
 .vscode/*
+
+# For sym internal testing
+sym.auto.tfvars
```

Comment:
The recommended override is `override.auto.tfvars`, which is already gitignored in this repo